tests/scaling_test.go

```diff
@@ -220,37 +226,106 @@ func Test_ScaleToZero(t *testing.T) {
 
 	defer deleteFunction(t, functionName)
 
+	attempts := 1000
```

Comment:
I am not sure `attempts` gels with me because we have retry code in the e2e tests. When I look at `hey -h` corresponds to `-n  Number of requests to run`. Would you be open to changing to `numReqs` / `totalReqs` or similar?